library/x509.c

```diff
@@ -701,22 +701,19 @@ int mbedtls_x509_get_sig_alg( const mbedtls_x509_buf *sig_oid, const mbedtls_x50
  * be either manually updated or extensions should be parsed!)
  */
 int mbedtls_x509_get_ext( unsigned char **p, const unsigned char *end,
-                  mbedtls_x509_buf *ext, int tag )
+                          mbedtls_x509_buf *ext, int tag )
 {
     int ret;
     size_t len;
 
-    if( *p == end )
-        return( 0 );
```

Comment:
We don't care from an API-stability perspective, because `mbedtls_x509_get_ext()`. We don't have a functional difference in our own usage of `mbedtls_x509_get_ext()`, because we don't call this function if `*p == end` as far as I see - calls to `mbedtls_x509_get_ext()` are pre-fixed by their own `*p == end` checks.

Reply:
No change.